python/servo/platform/macos.py

```diff
@@ -15,9 +15,9 @@
 from .. import util
 from .base import Base
 
-URL_BASE = "https://github.com/servo/servo-build-deps/releases/download/macOS"
-GSTREAMER_URL = f"{URL_BASE}/gstreamer-1.0-1.22.2-universal.pkg"
-GSTREAMER_DEVEL_URL = f"{URL_BASE}/gstreamer-1.0-devel-1.22.2-universal.pkg"
+URL_BASE = "https://github.com/servo/servo-build-deps/releases/download/macOS-2"
```

Comment:
Did you have to create a new tagged release because you were not able to edit the existing release and add the newer gstreamer packages? If so, I can just add it to the existing release as I'm able to edit them. I'll also look into how to give everyone access to edit the existing release.

Reply:
I didn't edit the release, because I thought there was a limit on the size of the release, but it appears this isn't the case. I can use the same release in a new version of this change.

Reply:
Okay. I've updated this change to use the same release.